File: FizzBuzz.c
#include <stdio.h>

// Print Fizz if divisble by 3
// Print Buzz if divisble by 5
// Print FizzBuzz if divisble by both.
// Otherwise print 1 - 100

int main(void) {
	int i;
	for(i = 0; i <= 100; i++) {		
		if (i % 3 == 0 && i % 5 == 0) {
			printf ("FizzBuzz: %d\n", i);
		} else if (i % 3 == 0) {
			printf ("Fizz: %d\n", i);
		} else if ((i%5) == 0) {
			printf ("Buzz: %d\n", i);
		} else	{ 
			printf("%d\n", i);
		}
	}	
	return 0;
}

File: FizzBuzz.py
#!/usr/bin/env python

# Jeremy Thimmesch, github.com/JeremyThimmesch
# FizzBuzz
# Print the numbers 1 - 100
# Print Fizz if divisible by 3, Buzz for 5, and FizzBuzz if both


for number in range(1,101):
    if number % 3 == 0 and number % 5 == 0:
        print number, "FizzBuzz"
    elif number % 3 == 0:
        print number, "Fizz"
    elif number % 5 == 0:
        print number, "Buzz" 
    else:
        print number
